xLights/TimingPanel.cpp

```diff
@@ -327,59 +327,59 @@ TimingPanel::TimingPanel(wxWindow* parent, wxWindowID id, const wxPoint& pos, co
     BitmapButton_Out_Transition_Adjust->SetLimits(OUT_TRANSITION_MIN, OUT_TRANSITION_MAX);
 
     // If i pull this from the map it sorts them alphabetically and I dont think that is what we want
-    Choice_LayerMethod->Append(_("Normal"));
```

Comment:
There is a comment here that specifically says that this is not what we want....  Transitions maybe, but not for the blending.

Reply:
But it just says "I dont think that is what we want".....  dont think... so not firm...
Why won't we want to make it easier to follow the list? It's not breaking anything, as far as I can tell. Had several users in the Zoom room when we discussed this and consensus was it makes sense to be sorted. Again, if it breaks something, I haven't yet found it.